lib/schemes/flat.js

```diff
@@ -6,5 +6,15 @@ module.exports = {
         var naming = bemNaming(options.naming);
 
         return naming.stringify(entity) + '.' + tech;
+    },
+    parse: function(path, options) {
+        options || (options = {});
+        var naming = bemNaming(options.naming),
+            splittedPath = path.split('.');
+
+        return {
```

Comment:
> Can't we refactor it later when we

Yes, we can.

> end up with some name

We decided to call it BemCell or BemCytus for now

> and found real cases for it?

https://github.com/bem-sdk/bem-decl/blob/master/lib/normalize.js#L13
https://github.com/bem-sdk/bem-graph/blob/master/lib/vertex.js#L7
etc.